alerting/src/alert-worker.ts

```diff
@@ -40,14 +44,19 @@ async function processAlert(alert: Alert, alertRepository: Repository<Alert>) {
   url.searchParams.append('deactivate', 'true');
   url.searchParams.append('email', email);
 
+  const urlWithParams = formatUrl(prismUrl, {
+    hazardLayerIds: serverLayerName,
+    date: maxDate.toISOString().slice(0, 10),
+  });
+
   if (alertMessage) {
     const emailMessage = `
         Your alert${alertName ? ` ${alertName}` : ''} has been triggered.
 
         Layer: ${serverLayerName}
```

Comment:
Let's use the `title` from the alertConfig instead of `serverLayerName` as the "Layer: ...."

Reply:
alertConfig don't have any `title`

type WCSConfig = {
  scale?: number;
  offset?: number;
  pixelResolution?: number;
};

export type AlertConfig = {
  type: string;
  serverLayerName: string;
  baseUrl: string;
  wcsConfig: WCSConfig;
};

what do you mean?

Reply:
Technically, the AlertConfig type is the same as `WMSLayerProps` that you can find in the website code. So it might be worth adding `title` and `id` for this usecase